doc/api/tls.md

```diff
@@ -898,6 +898,17 @@ It may be useful for debugging.
 
 See [Session Resumption][] for more information.
 
+### tlsSocket.isSessionReused()
+<!-- YAML
+added: v0.5.6
+-->
+
+* {boolean}
+
+Returns `true` if the session was reused, `false` otherwise.
```

Comment:
```suggestion
* Returns: {boolean} `true` if the session was reused, `false` otherwise.
```
(Line 906 can be removed, that format is used for properties, not methods.)

Reply:
I see now, I thought the comment was commentary on the suggested commit, I didn't see it was an additional change.

Reply:
Sorry, I should make a suggestion with the deleted line or be more clear.